client/src/components/ListGroup/ListGroup.scss

```diff
@@ -55,11 +56,16 @@
 
 .list-group-item__thumbnail {
   width: 64px;
-  height: 64px;
+  height: auto;
   display: block;
   background: $gray-lighter;
   float: left;
-  margin: -12px 12px 0 #{0 - $grid-gutter-width / 2};
+  margin: -12px 12px -12px #{0 - $grid-gutter-width / 2};
+
+  > img {
+    max-height: 65px;
+    height: 100%;
```

Comment:
It looks to me the car picture in the design is the case where it's resized and cropped to fill the designated space. I think stretching (aspect ration is respected) the image to fit is still better than having image different sizes. We can get Jared's input on this since Paul is away.